taps_to_cases/runner/src/main/java/com/microsoft/hydralab/t2c/runner/T2CJsonParser.java

```diff
@@ -46,7 +50,11 @@ private ArrayList<DriverInfo> getDriverList(JSONArray driverJsonArray) {
             String id = driverJsonObject.getString("id");
             String platform = driverJsonObject.getString("platform");
             driveIdToTypeMap.put(id, platform);
+            //Keep this to compat the old json format
             JSONObject initMassage = driverJsonObject.getJSONObject("init");
+            if (initMassage == null) {
+                initMassage = driverJsonObject.getJSONObject("setup");
```

Comment:
Please feel free to update the `setUp` to `setup` in `performance_test_design,puml` as well